modules/creating-control-plane-node.adoc

```diff
@@ -0,0 +1,387 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control_plane_machines_management/cpmso-manually-scaling-control-planes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-control-plane-node_{context}"]
+= Adding a control plane node to your cluster
+
+You can add 4 or 5 control plane nodes to the `ControlPlaneMachineSet` object that exists in your cluster. You can then verify the healthiness of the nodes. 
+
+The example in the procedure uses `node-5` as the new control plane node. 
+
+[IMPORTANT]
+====
+Red{nbsp}Hat supports only adding 4 or 5 control plane nodes to a cluster that runs on bare-metal infrastructure.
+====
+
+.Prerequisites
+
+* You have installed a healthy cluster with at least three control plane nodes.
+* You have created a single control plane node that you intend to add to your cluster as a postinstalltion task.
+
+.Procedure
+
+. Retrieve pending Certificate Signing Requests (CSRs) for the new control plane node by entering the following command:
++
+[source,terminal]
+----
+$ oc get csr | grep Pending
+----
+
+. Approve all pending CSRs for the control plane node by entering the following command:
++
+[source,terminal]
+----
+$ oc get csr -o go-template='{{range .items}}{{if not .status}}{{.metadata.name}}{{"\n"}}{{end}}{{end}}' | xargs --no-run-if-empty oc adm certificate approve
+----
++
+[IMPORTANT]
+====
+You must approve the CSRs to complete the installation.
+====
+
+. Confirm that the control plane node is in the `Ready` status by entering the following command:
++
+[source,terminal]
+----
+$ oc get nodes
+----
++
+[NOTE]
+====
+The etcd Operator requires a `Machine` custom resource (CR) that references the control plane node when the cluster runs with a Machine API. The Machine API automatically activates when the cluster has three or more control plane nodes.
```

Comment:
Machine API is enabled when you use an IPI installer, it's not available in UPI clusters unless retrofitted later.

Is the agent installer doing something special here that lead to this statement?